packages/@webex/plugin-meetings/test/unit/spec/breakouts/events.ts

```diff
@@ -30,17 +29,19 @@ describe('plugin-meetings', () => {
     });
     describe('postMoveCallAnalyzer', () => {
       it('send metric as expected', () => {
-        Metrics.postEvent = sinon.stub();
+        NewMetrics.submitClientEvent = sinon.stub();
         const eventInfo = {currentSession: newSession, meeting: mockMeeting, breakoutMoveId};
-        breakoutEvent.postMoveCallAnalyzer(eventType.BREAKOUT_JOIN_RESPONSE, eventInfo);
-        assert.calledWithMatch(Metrics.postEvent, {
-          event: eventType.BREAKOUT_JOIN_RESPONSE,
-          meetingId: 'activeMeetingId',
-          data: {
-            breakoutMoveId: 'breakoutMoveId',
-            breakoutSessionId: 'sessionId',
-            breakoutGroupId: 'groupId',
+        breakoutEvent.postMoveCallAnalyzer('client.breakout-session.join.response', eventInfo);
+        assert.calledWithMatch(NewMetrics.submitClientEvent, {
+          name: 'client.breakout-session.join.response',
+          payload: {
+            identifiers: {
+              breakoutMoveId: 'breakoutMoveId',
+              breakoutSessionId: undefined,
```

Comment:
why undefined values here? these are breakout unit tests so this doesn't feel right

Reply:
hmm, this is an actual bug! Thanks for catching it!
The previous implementation was broke, the parameters were not grabbed correctly from the eventInfo inside the source code.

Reply:
fixed now